src/test/java/seedu/address/testutil/TypicalReminders.java

```diff
@@ -0,0 +1,57 @@
+package seedu.address.testutil;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import seedu.address.model.ReminderAddressBook;
+import seedu.address.model.person.Name;
+import seedu.address.model.person.Reminder;
+
+/**
+ * A utility class containing a list of {@code Reminder} objects to be used in tests.
+ */
+public class TypicalReminders {
+
+    public static final Reminder MEETINGJASON = new Reminder("10-10-2022",
+                                                             "project meeting",
+                                                             new Name("Jason"));
+    public static final Reminder DINNERKATE = new Reminder("11-11-2022",
+                                                           "dinner",
+                                                           new Name("Kate"));
+    public static final Reminder BASKETBALLCARL = new Reminder("05-07-2023",
+                                                               "basketball practice",
+                                                               new Name("Carl"));
+    public static final Reminder JAPANTOM = new Reminder("12-12-2024",
+                                                         "Japan trip",
+                                                         new Name("Tom"));
+    public static final Reminder LUNCHFIONA = new Reminder("12-10-2024",
+                                                           "lunch",
+                                                           new Name("Fiona"));
+
+    // Manually added
+    public static final Reminder BREAKFASTLEON = new Reminder("13-10-2024",
```

Comment:
Hi, @nigellqx can I clarify what do you mean by manually added here? I thought this whole part of the code are reminders that are manually added?

Reply:
when u call getTypicalReminder(), it will return MEETINGJASON, DINNERKATE, BASKETBALLCARL, JAPANTOM, LUNCHFIONA put into the reminder address book. The BREAKFASTLEON and GYMTRISTAN is there to test for function such as adding of reminder to the reminder address book.